Sources/Calendar.php

```diff
@@ -365,6 +423,55 @@ function CalendarPost()
 	$context['linktree'][] = array(
 		'name' => $context['page_title'],
 	);
+
+	loadCSSFile('jquery-ui.datepicker.css', array('defer' => false), 'smf_datepicker');
+	loadCSSFile('jquery.timepicker.css', array('defer' => false), 'smf_timepicker');
+	loadJavascriptFile('jquery-ui.datepicker.min.js', array('defer' => true), 'smf_datepicker');
```

Comment:
In theory, changing the function's name to `loadJavaScriptFile` would be nice. But I'm sure there are many mods currently written to call `loadJavascriptFile` that would break if we changed it now.